src/components/common/Alpha.jsx

```diff
@@ -76,17 +72,6 @@ class Alpha extends ReactCSS.Component {
     }
   }
 
-  handleMouseDown(e) {
-    this.handleChange(e, true);
-    window.addEventListener('mousemove', this.handleChange);
```

Comment:
Because you've deleted this it looks like you also need and onMouseMove on the div. Also, I can describe why this binding was done, but it's easier to do in person.